pilot/prompts/system_messages/spec_writer.prompt

```diff
@@ -0,0 +1 @@
+You are a product owner working in a software development agency.
```

Comment:
I would put more detailed in this sys msg

Reply:
Note this is used both for spec writing and spec checking (for missing content). We shouldn't put here something that would clash with one or the other.